pages/linux/treetime.md

```diff
@@ -0,0 +1,20 @@
+# treetime
+
+> TreeTime provides routines for ancestral sequence reconstruction and inference of molecular-clock phylogenies.
+> More information: <https://manned.org/treetime>.
+
+- Infer ancestral sequences maximizing the joint or marginal likelihood:
+
+`treetime ancestral`
```

Comment:
I have no idea what any of this means, but what exactly happens when you run that command? Does it have to be executed inside a project? Does this print something? Does this open some interactive prompt?

Reply:
Yes it's supposed to print out this. Example: 
<img width="750" alt="image" src="https://user-images.githubusercontent.com/61142258/197024126-1ad6b52d-3db3-4483-bb59-16c56ab97f34.png">